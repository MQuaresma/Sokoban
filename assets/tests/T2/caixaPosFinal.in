###################
#####   ###########
#####   ###########
#####   ###########
###      ##########
### # ## ##########
#   # ## #####  ..#
#               ..#
##### ### # ##  ..#
#####     #########
###################
11 2
16 2
7 7
5 6
7 6
2 3
5 3