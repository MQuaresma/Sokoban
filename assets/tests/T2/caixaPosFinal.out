    #####
    #   #
    #   #
  ###  H##
  #  H H #
### # ## #   ######
#   # ## #####  ..#
# H  H          ..#
##### ### #o##  I.#
    #     #########
    #######
